vms/platformvm/dac/camino_proposal.go

```diff
@@ -62,6 +70,8 @@ type ProposalState interface {
 	IsSuccessful() bool // should be called only for finished proposals
 	Outcome() any       // should be called only for finished successful proposals
 	Visit(ExecutorVisitor) error
+	// Visits getter and returns additional lock tx ids, that should be unbonded when this proposal is successfully finished.
+	GetBondTxIDs(BondTxIDsGetter) ([]ids.ID, error)
```

Comment:
I'd leave it as it is - its always better to have descriptive naming
e.g.
`getter.Invoke()`
vs
`getter.GetBondTxIDs`

even if we replace getter with full bondTxIDsGetter, imo, its still better to have more descriptive func naming to be 100% clear about what is happening

Reply:
I'd leave it as it is - its always better to have descriptive naming
e.g.
`getter.Invoke()`
vs
`getter.GetBondTxIDs`

even if we replace getter with full bondTxIDsGetter, imo, its still better to have more descriptive func naming to be 100% clear about what is happening

Reply:
100% with you on descriptive naming
Don't get me wrong i'm all about functions expressing their intentions.

There's always though the danger of going too verbose. One has to always strike a balance between verbosity and readability.

In the same sense that within a Storage object you wouldn't call method initializeStorage() but rather initialize() because you can infer the rest from the context, you wouldn't invoke a getter function by saying get(Getter) but rather something like invoke/call Getter because that's what you do. You don't get a getter , you call it.

In a different context where the argument of this function wouldn't be such a detailed/explanatory interface,
Then invoke would be too vague and thus not recommended.

Imagine a method called 

GetAverage(amount1,amount2)
And another called
GetAverageOfTwoAmounts(amount1,amount2)

The latter is much more descriptive but crosses the lines of necessary verbosity.

If the argument types provide sufficient context about the function's purpose, you can opt for a shorter, more straightforward name. This promotes readability and maintains a clean and concise codebase.

Reply:
upd: was further discussed outside of gh, that lead to renaming of other dac visitor interfaces and corresponding methods